lib/cardano-sl.cabal

```diff
@@ -139,6 +157,24 @@ library
                         Pos.Statistics.Ekg
                         Pos.Statistics.Statsd
 
+                        Pos.DHT.Workers
```

Comment:
@avieth why these and few following modules were moved to `lib`? They were intentionally moved out of `lib` before to have finer code structure

Reply:
`Worker` is not defined in `cardano-sl-infra`, but in `cardano-sl`, because it depends upon the `Diffusion` type, which is defined in `cardano-sl`.